docs/notebooks/t-test.ipynb

```diff
@@ -39,7 +39,7 @@
    "cell_type": "markdown",
```

Comment:
<p>HDI for beta_1 has changed, just update the values.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>